.gitignore

```diff
@@ -87,3 +87,4 @@ GTAGS
 /modules
 *.swp
 compile_commands.json
+.idea/
```

Comment:
unrelated?

Reply:
Related to fix preparation.
It's a very minor change but if you insist it can be handled as a separate PR.

Reply:
should be another PR (update .gitignore)

Reply:
I'll put this into separate PR.